### DISCUSSION TITLE
Wazuh Agent on standalone is disconected

### DISCUSSION THREAD

**Discussion by sleepingbel** [Security-Onion-Solutions/securityonion#2033] [Unsupported Versions] [answered]
Hello,

Is it normal that the wazuh agent on the Security Onion is disconnected? The agent is running.

![Knipsel](https://user-images.githubusercontent.com/29328171/99879760-20b81a80-2c0f-11eb-9f44-53103278ee57.PNG)

It is a ISO standalone install.

Regards
Bart

**Comment by weslambert** [comment, marked as answer]
It should be running.  Are there any clues in the manager log in `/nsm/wazuh/logs/ossec.log`, or the agent log in `/var/ossec/logs/ossec.log`?

**Reply by sleepingbel** [reply]
Hello Wes,

Sorry, but with  upgrading to 2.3.10, SO was totaly broken and I installed it clean on sunday.
I think it was human mistake, me, in some of the configuration files.
You can close this Q&A

Regards 
Bart
